sparse/etc/pulse/x-maemo-route.table

```diff
@@ -1,2 +1,2 @@
-sink-input-by-media-role:x-maemo -15
-sink-input-by-media-role:phone -15 -28
+sink-input-by-media-role:x-maemo -25
+sink-input-by-media-role:phone -15
```

Comment:
First argument is default volume, second is minimum volume. If minimum volume is defined and a route with volume < minimum volume is activated, default volume is restored instead of the stored volume.

Minimum volume was useful when it was possible to completely mute one's voicecall, then when new call started the volume wouldn't be mute but default volume.